### ISSUE TITLE
Add "start" method

### ISSUE THREAD

**Issue by marcuspoehls** [supercharge/strings#40]
Add a new `.start(prefix)` method that prepends a single instance of the given `prefix` value to the start of the string if it doesn’t already start with the given `prefix`.

**Example**

```js
Str('repos/supercharge').start('/').get()
// '/repos/supercharge'

Str('/repos/supercharge').start('/').get()
// '/repos/supercharge'
```

**Comment by Jeconias** [comment]
You can assign it to me?

**Comment by marcuspoehls** [comment]
Resolved in #43
